### PR TITLE
Update the user guide

### PR DESCRIPTION
**Pull request by zhou-colla**
Update the sections related to the help window and the select command in the user guide

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -253,6 +249,7 @@ Action | Format, Examples
 **Batch-Delete**| `batch-delete t/TAG [t/TAG]...`<br> e.g. `batch-delete t/friends t/colleagues t/owesmoney t/...`
 **Batch-Edit**| `batch-edit t/OLDTAG t/NEWTAG`<br> e.g. `batch-delete t/friends t/frens`
 **Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
+**Find** | `find KEYWORD [MORE_KEYWORDS]...`<br> e.g., `find James Jake`
+**Select** | `select INDEX [MORE_INDEXES]...`<br> e.g., `find James Jake`
```
Comment (yyueyii) [inline docs/UserGuide.md:253]: The example for select command is find

**Review by yyueyii — CHANGES_REQUESTED** [review]
A minor issue, otherwise LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (zhou-colla) [reply docs/UserGuide.md]: Updated!
Change to : **Select** | `select INDEX [MORE_INDEXES]...`<br> e.g., `select 1 2`

**Review by yyueyii — APPROVED** [review]
lgtm

**Pull request merged**